gdx/src/com/badlogic/gdx/graphics/g3d/utils/CameraHelper.java

```diff
@@ -0,0 +1,215 @@
+			8, 17, 17, 18,
+			// Up triangle
+			19, 2, 2, 3, 3, 19};
+
+		int maxVertices = 20;
+		mesh = new Mesh(false, maxVertices, indices.length, new VertexAttribute(Usage.Position, 3, "a_position"),
+			new VertexAttribute(
+			Usage.ColorUnpacked, 4, "a_color"));
+		vertices = new float[maxVertices * (mesh.getVertexSize() / 4)];
+
+		mesh.setIndices(indices, 0, indices.length);
+	}
+
+	/** Update cameraHelper mesh. You should call this method if you update your camera. */
+	public void update () {
+		id = 0;
+		Vector3[] planePoints = camera.frustum.planePoints;
+
+		// 0 - 7 = Frustum points
+		for (int i = 0; i < planePoints.length; i++) {
+			vertice(planePoints[i], colorFrustum);
+		}
+
+		// 8 = Camera position
+		vertice(camera.position, colorCone);
+
+		// 9 - 12 = Cross near
+		vertice(middlePoint(planePoints[1], planePoints[0]), colorCross);
+		vertice(middlePoint(planePoints[3], planePoints[2]), colorCross);
+		vertice(middlePoint(planePoints[2], planePoints[1]), colorCross);
+		vertice(middlePoint(planePoints[3], planePoints[0]), colorCross);
+
+		// 13 - 16 = Cross far
+		vertice(middlePoint(planePoints[5], planePoints[4]), colorCross);
+		vertice(middlePoint(planePoints[7], planePoints[6]), colorCross);
+		vertice(middlePoint(planePoints[6], planePoints[5]), colorCross);
+		vertice(middlePoint(planePoints[7], planePoints[4]), colorCross);
+
+		// 17 - 18 = Target point
+		vertice(centerPoint(planePoints[0], planePoints[1], planePoints[2]), colorCross);
+		vertice(centerPoint(planePoints[4], planePoints[5], planePoints[6]), colorTarget);
+
+		// 19 = Up vertice
+		float halfNearSize = tmp.set(planePoints[1]).sub(planePoints[0]).scl(0.5f).len();
+		Vector3 centerNear = centerPoint(planePoints[0], planePoints[1], planePoints[2]);
+		tmp.set(camera.up).scl(halfNearSize * 2);
+		vertice(centerNear.add(tmp), colorUp);
+
+		// Set vertices
+		mesh.setVertices(vertices, 0, id);
+	}
+
+	/** Return the middle point of the segment
+	 * @param point0 First segment's point
+	 * @param point1 Second segment's point
+	 * @return the middle point */
+	private Vector3 middlePoint (Vector3 point0, Vector3 point1) {
+		tmp.set(point1).sub(point0).scl(0.5f);
+		return tmp2.set(point0).add(tmp);
+	}
+
+	/** Return the center point of the rectangle
+	 * @param point0
+	 * @param point1
+	 * @param point2
+	 * @return the center point */
+	private Vector3 centerPoint (Vector3 point0, Vector3 point1, Vector3 point2) {
+		tmp.set(point1).sub(point0).scl(0.5f);
+		tmp2.set(point0).add(tmp);
+		tmp.set(point2).sub(point1).scl(0.5f);
+		return tmp2.add(tmp);
+	}
+
+	private void vertice (Vector3 point, Color color) {
+		vertices[id++] = point.x;
+		vertices[id++] = point.y;
+		vertices[id++] = point.z;
+		vertices[id++] = color.r;
+		vertices[id++] = color.g;
+		vertices[id++] = color.b;
+		vertices[id++] = color.a;
+	}
+
+	@Override
+	public void getRenderables (Array<Renderable> renderables, Pool<Renderable> pool) {
+		Renderable r = pool.obtain();
```

Comment:
Since this class uses exactly one `Renderable` you might as well create it in the constructor and don't use the pool (or extend it, although I'm not sure that fits the scope of the usecase).